assets/scss/modules/_variables.scss

```diff
@@ -13,6 +13,13 @@ $shadow--darkmode: 0 8px 16px rgba(226, 226, 226, 0.1);
 $alert: #ffc107;
 $info: #0366d7;
 
+$notice-info--lightmode: rgb(239, 248, 255);
```

Comment:
I suggest building a more general theme where there is one color for info, warning and success respectively. Instead of having separate colors for each component, the general colors are reused. For its benefits, this is a widely adopted approach to theming, see: [MUI default theme](https://mui.com/customization/default-theme/), [Bulma Colors](https://bulma.io/documentation/overview/colors/).

Besides, we already 'introduced' this approach in #292 but currently then just had a need for the info color.

Reply:
@alexanderdavide Thanks for your feedback. I've drastically reduced the color variables by using the `mix` function that allowed me to blend the individual colors with the accent color of the respective theme mode. Would you mind having look at the most recent commit?

Reply:
This is great!